scripts/graphics/index.html

```diff
@@ -11,9 +25,27 @@
     var generator = new ActionGenerator();
     var model = new ViewModel();
 
-    generator.generateActions(model.updateModel);
+    generator.generateActions(function(action) {
+        logAction(action);
+        model.updateModel(action);
+    });
 
     var canvasViewer = new CanvasViewer(canvas, model);
     canvasViewer.start();
+
+
+    function logAction(action) {
+        var log = document.createElement('div');
+        log.innerText = JSON.stringify(action);
+        var logger = document.getElementById("log");
```

Comment:
if you refactor the JS file to be using RequireJS , you will be able to use Jquery for those DOM actions.